PWGHF/D2H/Macros/compute_fraction_cutvar.py

```diff
@@ -157,7 +200,9 @@ def main(config):
     hist_corry_nonprompt.Write()
     hist_covariance.Write()
     hist_corrfrac_prompt.Write()
-    hist_corrfrac_nonprompt.Write()
```

Comment:
I think that this has been deleted by mistake

Reply:
Hi @fgrosa, many thanks for spotting it! I have fixed it now